nss-day-cohort-48/the-worst-site-ever-deguello
# New landing page

## Steps to Test

1. Clone to local
2. Open in code and serve
3. Ensure that 1st site that loads is a white background with a click me link
4. Make sure the music autoplays and cant be stopped by the player in the top left.
